ml-agents/mlagents/trainers/ppo/multi_gpu_policy.py

```diff
@@ -50,6 +50,11 @@ def create_model(
                             PPOModel(
                                 brain=brain,
                                 lr=float(trainer_params["learning_rate"]),
+                                lr_schedule=LearningRateSchedule(
+                                    trainer_params.get(
+                                        "learning_rate_schedule", "linear"
```

Comment:
This should LearningRateSchedule.LINEAR, right? (same for the other places that it's used)

Reply:
I'm using the same convention that the `vis_encode_type` uses - maybe we can change both?

Reply:
Oh my bad, it's creating a new enum instance there. All good!